purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -2925,48 +2955,28 @@ class PurchasesTest {
                 observerMode = false,
                 subscriberAttributes = emptyMap(),
                 receiptInfo = productInfo,
+                storeAppUserID = null,
                 onSuccess = any(),
                 onError = any()
             )
         }
     }
 
     @Test
-    fun `when updating pending purchases, if result from query SUBS is not positive skip`() {
+    fun `when updating pending purchases, if result from querying purchases is not positive skip`() {
```

Comment:
I know this was here before, but should `is not positive skip` be `is not successful skip`?